src/main/java/edu/greenblitz/robotName/commands/Climb/lifter/ExtendLifter.java

```diff
@@ -0,0 +1,44 @@
+package edu.greenblitz.robotName.commands.Climb.lifter;
+
+import edu.greenblitz.robotName.subsystems.Lifter.LifterConstants;
+import edu.greenblitz.robotName.subsystems.Lifter.LifterSolenoid.LifterSolenoidConstants;
+import edu.wpi.first.wpilibj.Timer;
+
+public class ExtendLifter extends LifterCommand {
+    
+    protected Timer timer;
+
+    public ExtendLifter(){
+        timer = new Timer();
+    }
+    
+    @Override
+    public void initialize() {
+        lifterSolenoid.closeSolenoid();
+        timer.start();
+    }
+
+    @Override
+    public void execute() {
+        if (timer.hasElapsed(LifterSolenoidConstants.SECONDS_TO_CLOSE)){
+            lifterSolenoid.holdSolenoid();
+//            lifter.goToPosition(LifterConstants.LIFTER_EXTENDED_POSITION);
+            lifter.setPower(0.1);
+        }
+    }
+
+    @Override
+    public boolean isFinished() {
+//        return lifter.isAtPosition(LifterConstants.LIFTER_EXTENDED_POSITION);
```

Comment:
What? Delete comments if not real, but are you sure the command will never finish?